.yarnrc.yml

```diff
@@ -12,10 +16,4 @@ logFilters:
 
 nodeLinker: node-modules
 
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
```

Comment:
we want these plugins

Reply:
Double-checked and this is included by default in yarn 4. I think yarn pruned when i upgrade to latest version.

<img width="779" alt="image" src="https://github.com/user-attachments/assets/a85a2751-1e82-4db9-99a3-324fb39f8d1f">